pkg/proctree/proctree_feed.go

```diff
@@ -46,34 +46,43 @@ func (pt *ProcessTree) FeedFromFork(feed ForkFeed) error {
 
 	// Update the parent process (might already exist)
 
-	parent, ok := pt.GetProcessByHash(feed.ParentHash) // always a real process
-	if !ok {
-		parent = pt.GetOrCreateProcessByHash(feed.ParentHash)
+	setParentFeed := func(parent *Process) {
 		parent.GetInfo().SetFeedAt(
 			TaskInfoFeed{
+				Name:        "", // do not change the parent name
 				Tid:         int(feed.ParentTid),
 				Pid:         int(feed.ParentPid),
 				NsTid:       int(feed.ParentNsTid),
 				NsPid:       int(feed.ParentNsPid),
 				StartTimeNS: feed.ParentStartTime,
-				PPid:        -1, // do not change the parent PID
-				NsPPid:      -1, // do not change the ns parent PID
-				Uid:         -1, // do not change the parent UID
-				Gid:         -1, // do not change the parent GID
+				PPid:        0,  // do not change the parent ppid
```

Comment:
1. There exists pud number 0 (the sched task of the kernel), so why not using -1?
2. Since you have "parent" in the comment, should be parent pid and not parent ppid. Same for nsppid below

Reply:
1) yes, I believe all the values could be -1.
2) No, its parent ppid in this case. I'm not updating the parent ppid value.